mappings/net/minecraft/world/chunk/ChunkLoadingTask.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/unmapped/C_bdfidwdb net/minecraft/world/chunk/ChunkLoadingTask
+	METHOD doWork (Lnet/minecraft/unmapped/C_nclgamck;Lnet/minecraft/unmapped/C_amgmvhcu;Lnet/minecraft/unmapped/C_xgsefppr;Lnet/minecraft/unmapped/C_lwzmmmqr;)Ljava/util/concurrent/CompletableFuture;
```

Comment:
mojang's names for functional interfaces are so goofy. yeah that method *does* do work i guess

Reply:
Yeah... I'll change it to `run`

Reply:
oh huh, `doWork` was actually proposed, dunno where that came from

Reply:
it's obf, mojang leaves the names for functional interfaces in the jar for some reason

Reply:
ah ok